### PR TITLE
Feature/project6

### PR DESCRIPTION
**Pull request by GalOf-s**

### CODE REVIEW

File: 06/Main.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    Args:
+        input_file (typing.TextIO): the file to assemble.
+        output_file (typing.TextIO): writes all output to this file.
+    """
+    symbol_table = SymbolTable()
+
+    parser = Parser(input_file)
+    rom_address = 0
+
+    first_iteration = True
```
Comment (GalOf-s) [inline 06/Main.py]: why?

File: 06/Main.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        if parser.command_type() == L_COMMAND:
+            symbol_table.add_entry(parser.symbol(), rom_address)
+        else:
+            rom_address += 1
+
+    input_file.seek(0)
+    parser = Parser(input_file)
+    translated_lines = []
+
+    first_iteration = True
```
Comment (GalOf-s) [inline 06/Main.py]: what?

File: 06/Main.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    """Assembles a single file.
+
+    Args:
+        input_file (typing.TextIO): the file to assemble.
+        output_file (typing.TextIO): writes all output to this file.
+    """
+    symbol_table = SymbolTable()
+
+    parser = Parser(input_file)
+    rom_address = 0
```
Comment (GalOf-s) [inline 06/Main.py:27]: sure that's rom?

File: 06/Main.py (hunk 2 above)
Comment (amitkeinan1) [reply 06/Main.py]: first iteration of loop.
renamed and added comments.

File: 06/Main.py (hunk 3 above)
Comment (amitkeinan1) [reply 06/Main.py:27]: rom is the memory where the code is so I'm pretty sue it is

File: 06/Main.py (hunk 1 above)
Comment (amitkeinan1) [reply 06/Main.py]: I explained below in the other comment about it

**Pull request merged**